dist/tools/git/git-cache

```diff
@@ -217,6 +240,7 @@ usage() {
     echo "    git cache clone <url> <SHA1>  clone repository <url> from cache"
     echo "    git cache show-path           print's the path that can be used as "
     echo "                                  '--reference' parameter"
+    echo "    git cache cleanup             do some maintenance work"
```

Comment:
"some maintenance work"? Can you maybe be a bit more specific? From what I understand it removes dangling tags that do not have an associated commit anymore in the cache.

Reply:
Yes. An important upstream fix was the cleaning up of temprorary tags. That caused >10 seconds delay on Murdock job starts, after a couple of hundred checkouts. Now they're cleaned, unless the script is killed. For that case I introduced the manual cleanup, which should actually never be necessary to call...

Reply:
So can you amend the documentation?

Reply:
done!